modules/@apostrophecms/asset/lib/globalIcons.js

```diff
@@ -1,6 +1,8 @@
 // Alphabetical collection of globally available vue-material-design-icons.
 module.exports = {
+  'account-icon': 'Account',
```

Comment:
?

Reply:
I guess you are doing this for the convenience of the advanced permission module so it doesn't have to push separate icons. OK.

Reply:
Wait, that's not necessary. You can register this using the `icons` cascade in the npm module and it will still work. 

See the submitted-draft module for an example.

You don't have to add the material design icon module to the module to use this as long as it's a name that the core can find in the material design icon module.